packages/client/src/queries/applyQuery.ts

```diff
@@ -116,9 +116,9 @@ async function remapQueryResponse<
     const withoutMultiplicity = { ...responseDataType, multiplicity: false };
     for (let i = 0; i < responseValue.length; i++) {
       responseValue[i] = await remapQueryResponse(
-        responseValue[i],
-        withoutMultiplicity,
         client,
```

Comment:
This passed through fine before because `responseValue[i]` is typed any and that matched up with `MinimalClient` and the third param accepts type any (which is `DataValue`) so it took client :(

Reply:
Maybe we could use a branded type for these to force us to double check these / protect against refactors? Like `responseValue[i] as BrandedResponseValue` or whatever.

Reply:
Yea was thinking about that, can address in a FLUP